sdk-core/transactions-outcome-parsers/interfaces.md

```diff
@@ -0,0 +1,33 @@
+## Interfaces needed for parsing token management transactions
+
+```
+dto ITransactionEvent:
```

Comment:
If `dto`, without the prefix `I` (below, as well).

Reply:
removed